src/components/game-board/story-board/StoryBoard.tsx

```diff
@@ -49,11 +49,18 @@ export default function StoryBoard({className, updatePlayerTurn, game}: StoryBoa
         return !threeWordsPattern.test(activeText);
     }, [activeText]);
 
+    const isLast2Entries = mode === 'last_2_entries';
+
     return <div className={className}>
         <div className='flex flex-col h-3/4 w-full items-center'>
             <div className='text-container w-full h-3/4 flex flex-1 text-xl p-5'>
                 <div className='max-w-2xl'>
-                    {content}
+                    <span style={{filter: isLast2Entries && story.entries.length > 2 ? 'blur(4px)' : undefined}}>{story.opener}</span>
+                    {story.entries.map((entry, index) => (
```

Comment:
please notice that now story entries are not spaced between each other ,   before our last sync i hastingly copied the string building method in two placced like you can see in the game over , it prob should be the same function implemented in both places , but yeah , we want each entry to be spaced from the other